### PR TITLE
Offer toolbar redesign

### PR DESCRIPTION
**Pull request by hugokallstrom**
Add menu to offer toolbar, change title, and change background color according to new design

### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt [hunk 1]
```diff
@@ -64,13 +70,17 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
                     offerToolbar.setBackgroundColor(
                         boundedColorLerp(
                             Color.TRANSPARENT,
-                            compatColor(R.color.translucent_tool_bar),
+                            compatColor(R.color.colorBackground),
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt:73]:
```suggestion
                            colorAttr(android.R.attr.colorBackground),
```

This also might require an import.

File: app/src/main/res/layout/activity_offer.xml [hunk 2]
```diff
@@ -21,69 +22,18 @@
         android:id="@+id/offerToolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        app:navigationIcon="@drawable/ic_back"
         tools:layout_height="?attr/actionBarSize">
 
-        <LinearLayout
-            android:layout_width="match_parent"
+        <TextView
```
Comment (hedvigoscar) [inline app/src/main/res/layout/activity_offer.xml:28]: Should this be an `ImageView` of the Hedvig Wordmark? 🤔

**Review by hedvigoscar — CHANGES_REQUESTED** [review]
🚀

File: app/src/main/res/layout/activity_offer.xml (hunk 2 above)
Comment (hugokallstrom) [reply app/src/main/res/layout/activity_offer.xml:28]: Yeah I thought about that as well, do we have that as a resource already or should I export?

Comment (hugokallstrom) [reply app/src/main/res/layout/activity_offer.xml:28]: Found it :)

**Review by hedvigoscar — APPROVED** [review]

**Pull request merged**